packages/angular-workspace/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -152,21 +152,36 @@
             <div class="container-label">Select</div>
             <nimble-select filter-mode="standard" appearance="underline">
                 <nimble-list-option hidden selected disabled>Select an option</nimble-list-option>
```

Comment:
general question (don't need to change anything in example app), can your placeholder be inside a group? I'd imagine so but didn't look at test coverage.

Reply:
Yeah that should work, but no, I haven't added explicit tests around this as this wouldn't follow our guidance (we should update our guidance docs for this).

Reply:
From offline discussions sounds like we will add some tests to validate behavior so that does not block this PR.